.changesets/feat_local_safelist.md

```diff
@@ -0,0 +1,18 @@
+### Allow use of a local persisted query manifest for use with offline licenses ([Issue #4587](https://github.com/apollographql/router/issues/4587))
+
+This adds support to be able to pass [Persisted Query](https://www.apollographql.com/docs/graphos/operations/persisted-queries/) manifests to be used in place of the hosted Uplink version. 
+
+An example configuration would look like:
+
+```yml
+persisted_queries:
+  enabled: true
+  log_unknown: true
+  experimental_local_manifests: 
+    - ./persisted-query-manifest.json
+  safelist:
+    enabled: true
+    require_id: false
+```
```

Comment:
I know it's in the docs — and also that this PR is now already merged — but I might also change this changelog to lead with the "doesn't hot reload" gotcha.  Basically, this part, again:

> With the `experimental_local_manifests`, the router doesn't reload the manifest from the file system, so you need to restart the router to apply changes.